lib/widgets/transaction_header.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import '../theme/theme.dart';
+
+class TransactionHeader extends StatelessWidget {
+  final String value;
+  final VoidCallback? onSeeAllPressed;
+
+  const TransactionHeader({
+    super.key,
+    required this.value,
+    this.onSeeAllPressed,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Padding(
+      padding: EdgeInsets.symmetric(
+        horizontal: theme.sizing.width.s4,
+        vertical: theme.sizing.width.s2,
+      ),
+      child: Row(
+        children: [
+          Text(
+            (onSeeAllPressed != null) ? 'Transactions' : 'All Transactions',
+            style: theme.textStyle.titleRegular.copyWith(
+              color: theme.colors.contrastDark,
+            ),
+          ),
+          const SizedBox(width: 10),
```

Comment:
use theme; if 10 is not available try using 8 or 12 whichever suits better

Reply:
Done, used spacing and s10